src/Sulu/Bundle/ContentBundle/Resources/public/js/components/resource-locator/history/main.js

```diff
@@ -107,14 +107,14 @@ define(['text!./skeleton.html', 'text!./overlay.html'], function(skeletonTemplat
                         openOnStart: true,
                         removeOnClose: true,
                         instanceName: 'url-history',
-                        skin: 'medium',
+                        skin: 'large',
                         slides: [{
                             title: this.translations.title,
                             buttons: [{type: 'ok', align: 'center'}],
                             data: this.templates.overlay({
                                 translations: this.translations,
                                 histories: data,
-                                crop: this.sandbox.util.cropMiddle,
+                                crop: this.sandbox.util.cropFront,
```

Comment:
I think it's not a good idea to crop the front part, since this way you'll end up with a few exact same lines, if the URL of a parent has been changed, and it is not really visible...

I think the best solution would be to crop in the middle again, and the entire URL should be visible via a tooltip when hovering, or something like that.